src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -16,21 +19,27 @@
 
     @Test
     public void testIsCloudStorageURL(){
-        Assert.assertTrue(BucketUtils.isCloudStorageUrl("gs://a_bucket/bucket"));
+        Assert.assertTrue(BucketUtils.isCloudStorageUrl("gs://abucket/bucket"));
```

Comment:
That's an interesting story. Underscore is [legal](https://cloud.google.com/storage/docs/naming) in bucket names, but the same documentation strongly discourages using them because they are not legal in host names. The second element of URIs is normally a host name, so URI.create will fail for a URI like "gs://a_bucket/file". It [appears](http://www.zytrax.com/books/dns/apa/names.html) that underscores are now legal in host names, but they are still not accepted by the URI constructor.